olive/systems/system_config.py

```diff
@@ -147,7 +147,15 @@ def validate_config_type(cls, values):
         if system_alias_class:
             values["type"] = system_alias_class.system_type
             if system_alias_class.accelerators:
-                values["config"]["accelerators"] = [{"device": acc} for acc in system_alias_class.accelerators]
+                valid_accelerators = []
+                for device in system_alias_class.accelerators:
+                    valid_accelerators.extend(
+                        {"device": acc["device"], "execution_providers": acc.get("execution_providers")}
```

Comment:
If acc is not none, `execution_providers` must exist, right?

Reply:
acc cannot be None for aml, so do as ep